javascripts/discourse/components/topic-in-gated-category.js

```diff
@@ -26,18 +29,20 @@ export default Component.extend({
 
   recalculate() {
     // do nothing if:
-    // a) topic does not have a category
+    // a) topic does not have a category and does not have a gated tag
     // b) component setting is empty
     // c) user is logged in
+    this.gatedByTag = this.tags.some((t) => enabledTags.indexOf(t) >= 0);
```

Comment:
Functionality is the same, but I think I prefer the readability of
```suggestion
    this.gatedByTag = this.tags.some((t) => enabledTags.includes(t));
```